spec/spec_helper.rb

```diff
@@ -37,6 +37,11 @@ def require(path)
   end
 end
 
+class Formatter
```

Comment:
Yeah, just wanted it to be consistent with what's in core. Out of curiosity, why exactly is this being stubbed?